src/actions/AddSiteActions.js

```diff
@@ -12,6 +12,7 @@ import {
     SITE_NEAREST_TOWN_TEXT_CHANGED,
     SITE_ACCESSIBILITY_OPTION_CHANGED,
     SITE_FACILITIES_OPTION_CHANGED,
+    SITE_PRICE_OPTION_CHANGED,
     ADD_SITE_FIELDS_RESET,
```

Comment:
I added an option to select `price` level in `AddSiteScreen`, so I had to also add the corresponding type, action, and reducer to handle it being passed around.